editoast/fga/src/client.rs

```diff
@@ -273,9 +280,10 @@ impl Client {
         Ok(model_id)
     }
 
-    /// Writes up to 100 tuples in OpenFGA
+    /// Writes up to n tuples in OpenFGA, with n the maximum number of tuple writes
```

Comment:
nit

```suggestion
    /// Writes up to `n` tuples in OpenFGA, with `n` the maximum number of tuple writes
```

and so on